lib/ex_aws/request/http_client.ex

```diff
@@ -20,11 +20,21 @@ defmodule ExAws.Request.HttpClient do
   ```elixir
   defmodule ExAws.Request.HTTPotion do
     @behaviour ExAws.Request.HttpClient
-    def request(method, url, body, headers) do
+    def request(method, url, body, headers, http_opts) do
       {:ok, HTTPotion.request(method, url, [body: body, headers: headers, ibrowse: [headers_as_is: true]])}
     end
   end
   ```
+
+  When conforming your selected HTTP Client take note of a few things:
+    - The module name doesn't need to follow the same styling as this module it is simply your own 'HTTP Client', i.e. #{project_name}.HttpClient
```

Comment:
@benwilson512  compile error project_name does not exists

Reply:
`#{project_name}` is string interpolation so you would replace that with whatever your project is called. 
At the top level of your project you usually have a module with the same name as your project:
```elixir
defmodule MyApp do
end
``` 
You could then have a folder structure similar to this:
my_app.ex
clients
|-> http_client.ex
inside your http_client.ex you would namespace it with your app name:
```elixir
defmodule MyApp.HttpClient do
end
```
In this case `MyApp` is what replaces the `#{project_name}` here.

Reply:
Sorry it took so long for a reply!